ui/app/AppLayouts/Communities/popups/HoldingsDropdown.qml

```diff
@@ -324,7 +326,16 @@ StatusDropdown {
                 if(d.extendedDropdownType === ExtendedDropdownContent.Type.Assets)
                     root.assetKey = key
                 else
+                {
                     root.collectibleKey = key
+                    const amount = PermissionsHelpers.getTokenAmountByKey(root.collectiblesModel, root.collectibleKey)
+                    if(amount == 1) {
```

Comment:
Maybe you can add here a comment to be more understandable that when there is a unique collectible calling to these `update or add` signals what really does is a jump in navigation!